gatsby-config.js

```diff
@@ -29,7 +29,7 @@ module.exports = {
 				background_color: '#fff',
 				theme_color: '#c960ff',
 				display: 'minimal-ui',
-				icon: 'src/images/gatsby-icon.png' // This path is relative to the root of the site.
```

Comment:
You should remove this file.

Reply:
Oh! Good catch, thank you. Resolved in the next commit!